### PR TITLE
Configure repository

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we configure this repository in consistency with all other Spine Event Engine repositories.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Comment by armiol** [comment]
As discussed, let's make this library JDK7-compliant.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
+        testCompile("io.spine:spine-testutil-base:${spineBaseVersion}") {
+            // We are using JUnit 5.x instead of 4.x dependency of Spine testing framework.
+            exclude group: 'junit'
+        }
+        testCompile("io.spine:spine-testutil-core:${spineVersion}") {
+            // We are using JUnit 5.x instead of 4.x dependency of Spine testing framework.
+            exclude group: 'junit'
+        }
+        testCompile("io.spine:spine-testutil-client:${spineVersion}") {
```
Comment (armiol) [inline build.gradle]: This dependency looks redundant.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+    apply plugin: 'com.google.protobuf'
+    apply plugin: 'org.junit.platform.gradle.plugin'
+    apply plugin: 'io.spine.tools.spine-model-compiler'
+
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
+
+    dependencies {
+        compile (group: 'io.spine', name: 'spine-testutil-server', version: spineVersion)
```
Comment (armiol) [inline build.gradle]: This dependency looks redundant.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        compile group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
+        compile group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.0'
+        compile group: 'com.google.guava', name: 'guava', version: guavaVersion
+
+        testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
+        testCompile("io.spine:spine-testutil-base:${spineBaseVersion}") {
+            // We are using JUnit 5.x instead of 4.x dependency of Spine testing framework.
```
Comment (armiol) [inline build.gradle]: Are we using JUnit5 with JDK7? I don't think so.
  
Same below. For several more times.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+language: java
+
+jdk:
+  - oraclejdk8
```
Comment (armiol) [inline .travis.yml]: Each time we talk about that. Can we have a comment explaining why JDK7-compatible project is built in JDK8-powered environment?

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov please see my comments.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

**Review by armiol — APPROVED** [review]
@dmdashenkov LGTM. I also expect to see publishing configured somehow.

**Pull request merged**